source/templates.tex

```diff
@@ -7136,12 +7131,13 @@
 otherwise, it is a declaration.
 \begin{note}
 The definition of a static data member of a template
-that requires default-initialization must use a \grammarterm{braced-init-list}:
+that requires default-initialization
```

Comment:
```suggestion
for which default-initialization is desired
```
(Just a drive-by to avoid using "requires" where we're not specifying a requirement.)